Aleksey Vlaskin/index.js

```diff
@@ -1,14 +1,112 @@
-const ansi = require('ansi');
+const readline = require('readline');
 
-const cursor = ansi(process.stdout);
+const fs = require('fs');
 
-cursor
-    .red()
-    .bg.yellow()
-    .write('Hello!!!')
-    .bg.reset()
-    .reset()
-    .write('\n');
-    
-cursor.beep();
+const argv = require('minimist')(process.argv.slice(2));
 
+if (argv.a) {
+    try {
+        fs.readFile(argv._[0], 'utf-8', (err, data) => {
+            if (err) throw err;
+                        
+            let arrData = data.split(',').filter(elem => {return elem!=='';}); //уберём пустой элемент после окончания игры, если он есть
+            const total = arrData.length;
+            console.log('games:', total);
+             
+            if (total ===0) {return};
+            
+            let arrWin = arrData.filter(elem => {return elem === '1'});
+            let arrDef = arrData.filter(elem => {return elem === '0'});
+            
+            const wins = arrWin.length;
+            const defs = arrDef.length;
+                    
+            const persWins = Math.round(parseFloat(wins/total*100) * 100) / 100;
+            
+            console.log('wins:', wins + ',', Math.round(parseFloat(wins/total*100) * 100) / 100 + '\%');
+            console.log('defs:', defs + ',', 100-persWins + '\%');
+            
+            let maxWinsRow = 0;
+            let maxDefsRow = 0;
+            let nowWin = 0;
+            let nowDef = 0;
+            
+            for (i=0; i<total; i++){
```

Comment:
Будьте внимательнее с оформлением кода. Рекомендую расставить пробелы между операторами и операндами
```suggestion
            for (i=0; i < total; i++) {
```

Reply:
По подмене persWins и лишнему firstGame - увидел уже после создания пуллреквеста, не стал переделывать. По оформлению - стараюсь пробелы ставить, в 1С за 10 лет привык форматировать красиво, чтоб читать было приятно, здесь же периодически плагин Beautify на Brackets оформляет не совсем так, как хотелось бы. Постараюсь следить!